career-mentorship-program/2021-spring/matching_code.py

```diff
@@ -175,6 +194,21 @@ class Mentee(Student):
     # Which areas of the recruitment process the mentee wants help with
     desired_topics: Set[str]
 
+    # Hours per week the mentor is able to dedicate
+    available_time: int
+
+    # What are their hobbies/interests
+    hobbies: Set[str]
+
+    # if prefer gender
```

Comment:
Let's use a more descriptive comment here to show that means the mentee/mentor wants to be matched with someone of the same gender.